### PR TITLE
Replace SQLite with actual database connection

### PR DESCRIPTION
**Pull request by MattW-source**
Introduces new requirement/dependency: **sql**

### CODE REVIEW

**Comment by MM-coder** [comment]
Seems good, no blatant worries or security concerns

File: commands/cosmetics.py [hunk 1]
```diff
@@ -15,9 +15,9 @@ def __init__(self, client):
     async def cosmetics(self, ctx, operand = None, cosmetic = None):
         if operand is None:
             #get inventory
-            cosmetics_all = sql.db_query("ibm.db", "SELECT cosmetic_id FROM cosmetics")
+            cosmetics_all = sql.db_query("SELECT cosmetic_id FROM cosmetics")
             cosmetics_total = len(cosmetics_all)
-            inventory = eval(sql.db_query("ibm.db", "SELECT cosmetics FROM Members WHERE UserID = %s" % (str(ctx.author.id)))[0][0])
+            inventory = eval(sql.db_query("SELECT cosmetics FROM Members WHERE UserID = %s" % (str(ctx.author.id)))[0][0])
```
Comment (MM-coder) [inline commands/cosmetics.py:20]: Maybe use sql COUNT instead of eval here

File: commands/cosmetics.py [hunk 2]
```diff
@@ -35,14 +35,14 @@ async def cosmetics(self, ctx, operand = None, cosmetic = None):
 
         elif operand.upper() == "SHOW" or operand.upper() == "DISPLAY":
             index = int(cosmetic)
-            inventory = eval(sql.db_query("ibm.db", "SELECT cosmetics FROM Members WHERE UserID = %s" % (str(ctx.author.id)))[0][0])
+            inventory = eval(sql.db_query("SELECT cosmetics FROM Members WHERE UserID = %s" % (str(ctx.author.id)))[0][0])
```
Comment (MM-coder) [inline commands/cosmetics.py:38]: Same eval replacement here

File: commands/profile.py [hunk 3]
```diff
@@ -24,7 +24,7 @@ async def profile(self, ctx, user : discord.Member = None):
         else:
             em = discord.Embed(title=user.name, colour=eval(profile[4]))
```
Comment (MM-coder) [inline commands/profile.py:25]: Why is there even an eval call here?

**Review by MM-coder — COMMENTED** [review]
Too many eval calls

File: commands/profile.py (hunk 3 above)
Comment (MattW-source) [reply commands/profile.py:25]: This is out of scope for this PR. Please raise an issue instead.

File: commands/cosmetics.py (hunk 2 above)
Comment (MattW-source) [reply commands/cosmetics.py:38]: This is out of scope for this PR. Please raise an issue instead.

File: commands/cosmetics.py (hunk 1 above)
Comment (MattW-source) [reply commands/cosmetics.py:20]: This is out of scope for this PR. Please raise an issue instead.

**Review by TrueParrot — APPROVED** [review]
Looks good

**Pull request merged**